init.c

```diff
@@ -99,44 +99,45 @@ struct MyVar
 {
   char *name;
   char *value;
-  struct MyVar *next;
+  TAILQ_ENTRY(MyVar) entries;
 };
 
-static struct MyVar *MyVars;
+TAILQ_HEAD(MyVarList, MyVar);
+static struct MyVarList MyVars = TAILQ_HEAD_INITIALIZER(MyVars);
```

Comment:
Oh, btw, it's also possible to have the head as an anonymous struct..
```c
static TAILQ_HEAD(, MyVar) MyVars = TAILQ_HEAD_INITIALIZER(MyVars);
```

Reply:
Done.

Reply:
Thanks!